azure-pipelines.yml

```diff
@@ -1,20 +1,20 @@
 jobs:
-- job: BuildLinuxWithSonarCloud
-  displayName: Build for Linux\CMake With SonarCloud
+- job: BuildLinuxWithSonarQube
+  displayName: Build for Linux\Autotools With SonarQube
   pool:
     vmImage: 'ubuntu-latest'
-  variables:
-    BUILD_WRAPPER_DOWNLOAD_URL: https://sonarcloud.io/static/cpp/build-wrapper-linux-x86.zip
   steps:
-  - task: SonarCloudPrepare@1
+  # Make Build Wrapper available
+  - task: SonarQubePrepare@4
     inputs:
-      SonarCloud: 'SonarCloud' # Name of the SonarCloud service connection you created
-      organization: 'sc-sq-c-family-examples'
+      SonarQube: 'azure-sq' # Name of the SonarQube service endpoint you created
       scannerMode: 'CLI'
       configMode: 'file'
+      extraProperties: "sonar.cfamily.build-wrapper-output=bw_output"
   - bash: |
       mkdir -p $HOME/.sonar
-      curl -sSLo $HOME/.sonar/build-wrapper-linux-x86.zip ${{variables.BUILD_WRAPPER_DOWNLOAD_URL}}
+      # Replace <SONARQUBE_HOST_URL> with your SonarQube server URL
+      curl -sSLo $HOME/.sonar/build-wrapper-linux-x86.zip '<SONARQUBE_HOST_URL>/static/cpp/build-wrapper-linux-x86.zip'
```

Comment:
What it be possible to have SONAR_QUUBE_HOST_URL as the variable, I think it will make it cleaner, what part need to be repalced.